humanize-duration.js

```diff
@@ -490,7 +490,14 @@
 
       // What's the number of full units we can fit?
       if (i + 1 === len) {
-        unitCount = ms / unitMS
+        if (options.maxDecimalPoints) {
+          // We need to use this expValue to avoid rounding functionality of toFixed call
+          var expValue = Math.pow(10, options.maxDecimalPoints)
+          var unitCountFloat = (ms / unitMS)
+          unitCount = parseFloat((Math.floor(expValue * unitCountFloat) / expValue).toFixed(options.maxDecimalPoints))
```

Comment:
We could reorder this a little bit, I think:

```javascript
// What's the number of full units we can fit?
unitCount = ms / unitMS
if ((i + 1 === len) && options.hasOwnProperty('maxDecimalPoints')) {
  // We need to use this expValue to avoid rounding functionality of toFixed call
  var expValue = Math.pow(10, options.maxDecimalPoints)
  unitCount = parseFloat((Math.floor(expValue * unitCount) / expValue).toFixed(options.maxDecimalPoints))
} else {
  unitCount = Math.floor(unitCount)
}
```

Shortens things a bit.

Reply:
Since I'm a bit out of context right now, I'd prefer if I keep the code this way. Feel free to change the code afterward. The code you've posted doesn't really work the same way.